src/overlay.css

```diff
@@ -32,21 +33,21 @@
   background-color: white;
   z-index: 100;
   color: black;
-  padding: 1em;
```

Comment:
How does this change affect the overlays? Like the state editor and code editors